src/test/java/smokefree/projection/InitiativeProjectionTest.java

```diff
@@ -121,8 +123,13 @@ InitiativeCreatedEvent initiativeCreated(Status status) {
         return initiativeCreated(UUID.randomUUID().toString(), status);
     }
 
+    private static final String PLAYGROUND_NAME_INITIATIVE_1 = "Happy Smokefree";
```

Comment:
We need to discuss on what kinds of unit tests we do. I would prefer starting from commands or even the controllers and validating the resulting projections. This would allow us to get better code coverage in an efficient way